tensorflow/g3doc/get_started/os_setup.md

```diff
@@ -859,7 +859,7 @@ and you left the Cuda or cuDNN version empty, try specifying them explicitly.
 ### Protobuf library related issues
 
 TensorFlow pip package depends on protobuf pip package version
-3.0.0b2. Protobuf's pip package downloaded from [PyPI](https://pypi.python.org)
+3.0.0. Protobuf's pip package downloaded from [PyPI](https://pypi.python.org)
```

Comment:
as far as I see the protobuf pip package is still at 3.0.0. Or am I missing something?